src/js/events.js

```diff
@@ -137,6 +137,21 @@ define(function (require, exports, module) {
             LOAD_HISTORY_STATE_REVERT: "loadHistoryStateRevert",
             ADJUST_HISTORY_STATE: "adjustHistoryState",
             DELETE_DOCUMENT_HISTORY: "deleteDocumentHistory"
+        },
+        droppable: {
```

Comment:
I think this is a rebase error. All the droppable events have been removed in favor of direct store access.